plan/entities.go

```diff
@@ -55,11 +58,37 @@ func (a Axes) Index() map[string]Axis {
 	return axes
 }
 
-// Behavior represents the test behavior that will be triggered by crossdock
+// Filter specifies criteria for skipping specific test cases of a behavior.
+// All test cases for a behavior where all parameter values match the AxisMatches will be skipped.
+type Filter struct {
+	AxisMatches []AxisMatch
```

Comment:
Can we rename this attribute to Matchers?

(This doesn't have to be manual. Your editor should take care of it for you.)